### PR TITLE
Create response to slap in bear/bear.md - slap-response/slap-response.md

### PR DESCRIPTION
**Pull request by Vladimir-Markovic**

### CODE REVIEW

**Comment by Vladimir-Markovic** [comment]
Please review this, @casey-collab

File: english/slap-response/slap-response.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"urasinner!"; smile and add "aren't we all", turning your other cheek.
```
Comment (casey-collab) [inline english/slap-response/slap-response.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Vladimir-Markovic** [comment]
Please review this, @casey-collab

File: english/slap-response/slap-response.md (hunk 1 above)
Comment (casey-collab) [inline english/slap-response/slap-response.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Vladimir-Markovic** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**